packages/examples/package.json

```diff
@@ -1,28 +1,30 @@
 {
-  "name": "examples",
+  "name": "@metamask/example-snaps",
   "version": "0.34.1-flask.1",
   "private": true,
-  "description": "Example MetaMask Snaps.",
   "repository": {
     "type": "git",
     "url": "https://github.com/MetaMask/snaps.git"
   },
-  "license": "MIT",
-  "files": [
-    "examples/"
-  ],
+  "files": [],
   "workspaces": [
-    "examples/*"
+    "packages/*"
   ],
   "scripts": {
-    "lint:changelog": "yarn auto-changelog validate",
+    "build": "yarn workspaces foreach --parallel --verbose --no-private run build",
```

Comment:
Why are we using `--no-private`?

Reply:
The `packages/invoke-snap` folder is a separate (private) workspace, where we do not want to run `build` directly, but rather run it on the packages inside that workspace.